application/models.py

```diff
@@ -34,6 +34,7 @@ class HighSchoolApplication(models.Model):
     program = models.CharField(max_length=100)
     is_draft = models.BooleanField(default=True)
     submitted_date = models.DateTimeField()
+    application_status = models.CharField(default="pending", max_length=10)
```

Comment:
We might have a more robust model if we chose the application_status field to be an integer instead of a char field. This has the following advantages - 
- there's quicker query time in comparing values against integers than strings (this might not be too much for our app, but still)
- it helps limit any unintended typos (0 - rejected, 1 - accepted, 2 - pending). 
This might be considered a nit.

Reply:
this should be similar to gender list.

Reply:
> this should be similar to gender list.
solved